src/app/app.module.ts

```diff
@@ -25,12 +31,14 @@ import { Api } from '../providers/api';
 import { Settings } from '../providers/settings';
 import { Items } from '../mocks/providers/items';
 
-import { TranslateModule, TranslateLoader, TranslateStaticLoader } from 'ng2-translate/ng2-translate';
+import { TranslateModule, TranslateLoader } from '@ngx-translate/core';
+import { TranslateHttpLoader } from '@ngx-translate/http-loader';
+//TranslateStaticLoader
 
 // The translate loader needs to know where to load i18n files
 // in Ionic's static asset pipeline.
 export function createTranslateLoader(http: Http) {
-  return new TranslateStaticLoader(http, './assets/i18n', '.json');
+  return new TranslateHttpLoader(http, './assets/i18n', '.json');
```

Comment:
You are correct:

> replace the TranslateStaticLoader ()if you use it) with the TranslateHttpLoader

https://github.com/ngx-translate/core/blob/master/MIGRATION_GUIDE.md#migration-guide-from-ng2-translate-5-to-ngx-translate-6